<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_agregar_citas</class>
 <widget class="QDialog" name="Dialog_agregar_citas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>375</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog_agregar_citas</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AGREGAR CITA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>241</width>
     <height>282</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>USUARIO DEL PACIENTE:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="insertar_usuariopac"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>ESPECIALIDAD:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="insertar_espec"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>TURNO:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="insertar_turnM">
      <property name="text">
       <string>Ma√±ana</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="insertar_turnT">
      <property name="text">
       <string>Tarde</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>FECHA:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="insert_fecha"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QPushButton" name="agregar">
      <property name="text">
       <string>AGREGAR</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="volver">
      <property name="text">
       <string>VOLVER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
